plugin/src/main/java/net/dzikoysk/funnyguilds/Entity.java

```diff
@@ -14,4 +17,10 @@
 
     String getName();
 
+    static <T extends Entity> List<String> names(Iterable<T> entities) {
```

Comment:
Myślę, że Set wystarczy. Po co Lista?

Reply:
Zachowanie kolejności elementów

Reply:
Ale właściwie po co zachowywać ich kolejność 🤔 To jest chyba używane w miejscach gdzie i tak mogą być one użyte w jakiekolwiek kolejności...

Reply:
no w sumie i tak wszystko jest w tych setach